tutorials/tour/_posts/2017-02-13-sequence-comprehensions.md

```diff
@@ -13,58 +13,39 @@ previous-page: extractor-objects
 
 Scala offers a lightweight notation for expressing *sequence comprehensions*. Comprehensions have the form `for (enumerators) yield e`, where `enumerators` refers to a semicolon-separated list of enumerators. An *enumerator* is either a generator which introduces new variables, or it is a filter. A comprehension evaluates the body `e` for each binding generated by the enumerators and returns a sequence of these values.
 
-Here is an example:
- 
+Here's an example:
+
 ```tut
-object ComprehensionTest1 extends App {
-  def even(from: Int, to: Int): List[Int] =
-    for (i <- List.range(from, to) if i % 2 == 0) yield i
-  Console.println(even(0, 20))
-}
-```
- 
-The for-expression in function introduces a new variable `i` of type `Int` which is subsequently bound to all values of the list `List(from, from + 1, ..., to - 1)`. The guard `if i % 2 == 0` filters out all odd numbers so that the body (which only consists of the expression i) is only evaluated for even numbers. Consequently, the whole for-expression returns a list of even numbers.
+case class User(val name: String, val age: Int)
 
-The program yields the following output:
+val userBase = List(new User("Travis", 28),
+  new User("Kelly", 33),
+  new User("Jennifer", 44),
+  new User("Dennis", 23))
```

Comment:
Although the style of indentation in parens varies widely, I'm not sure this one is a good one for a tutorial since we don't do trailing braces in Scala, generally, and trailing parens aren't that different from trailing braces.  I'd format it as
```scala
val userBase = List(
  new User("Travis", 28),
  ...
  new User("Dennis", 23)
)
```